### ISSUE TITLE
Search: Buttons for removing filters should be labelled properly

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#122]
URL: https://hcdev.openplus.ca/en/search/site
User Agent: Chrome
Language: Both

Issue Description: The "Clear all" button has title attribute, hidden text and visual text for its label. Screen readers read it as "Remove Clear all button clear all filters". See code below:
`<a href="https://hcdev.openplus.ca/en/search/site" title="Clear all filters" class="btn btn-clear-all ndm-facet-button-close display-flex flex-align-center" role="button">
      <span class="wb-invisible">Remove</span>Clear all
      <svg class="svg-inline--fa fa-times-circle fa-w-16 pl-1" role="presentation" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="times-circle" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" data-fa-i2svg=""><path fill="currentColor" d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"></path></svg>
    </a>`
For the other buttons, screen readers only read as something like "Type: Health product recall button" and "Audience: General public button". Screen reader users will not know they can remove that filter if they activate that button.
![image](https://user-images.githubusercontent.com/88776404/137165340-2287e0f6-027b-4a5c-96a5-4917ae8ae30c.png)

Steps to reproduce:

1. Navigate to the Search page https://hcdev.openplus.ca/en/search/site
2. Selecting some filters from the right hand side
3. Turn on screen readers and navigate to the "Search" part
4. Listen the buttons.

Expected results: "Clear all" button should be read as "Clear all filters button". The other buttons should be read as something like this "Remove Type: Health product recall filter button".

**Comment by echouin** [comment]
@XiuJuanXia resolved.

**Comment by ndolter** [comment]
Validated that the screen reader (JAWS) now provides instructions on the various buttons and reads 'Remove filter for <button name>. To activate press enter.'
